### ISSUE TITLE
Question on converting okteto v1 manifest to v2 manifest

### ISSUE THREAD

**Issue by Zionett** [okteto/remote-kubernetes#189]
Is there a guide on how I can convert my v1 manifest to v2 with a pre-built image? The extension upgraded my okteto CLI so now the old manifest is not working

**Comment by rberrelleza** [comment]
The b1 of manifest should still work. What error are you getting?

**Comment by Zionett** [comment]
This is what I get. FWIW I already have `autocreate: true` in `okteto.yaml` and it was working before upgrading okteto to 2.1.0
```
 !  The field 'annotations' is deprecated and will be removed in version 2.2.0. Use the field 'metadata.annotations' instead (https://okteto.com/docs/reference/manifest/#metadata)
 i  Using <context> as context
 !  The field 'annotations' is deprecated and will be removed in version 2.2.0. Use the field 'metadata.annotations' instead (https://okteto.com/docs/reference/manifest/#metadata)
 i  Images were already built. To rebuild your images run 'okteto build' or 'okteto deploy --build'
 x  Application '<application>' not found in namespace '<namespace>'
    Verify that your application is running and your okteto context is pointing to the right namespace
    Or set the 'autocreate' field in your okteto manifest if you want to create a standalone development container
    More information is available here: https://okteto.com/docs/reference/cli/#up
```

**Comment by Zionett** [comment]
FWIW I already have `autocreate: true` in `okteto.yaml` and it was working before upgrading okteto to 2.1.0
```
name: <name>
annotations:
  iam.amazonaws.com/role: <awsrole>
context: <context>
namespace: <namespace>
image: <image>
autocreate: true
workdir: <workdir>
sync:
  - .:<workdir>
command: bash
resources:
  limits:
    memory: "16Gi"
    cpu: "4"
persistentVolume:
  enabled: false
```

**Comment by rberrelleza** [comment]
this should be fixed on 0.4.2. Could you try it out and reopen if it still happens?
